Documentation.md

```diff
@@ -39,24 +39,34 @@ To whitelist your pages:
 * All of the pages that attempt to initialize the Streams library must be hosted
   on domains that are whitelisted as per the previous steps.
 * All open tabs that contain an initialized Streams library or any other CCP
-  tabs opened will be synchronized.  This means that state changes made in one
+  tabs opened will be synchronized. This means that state changes made in one
   open window will be communicated to all open windows.
 * Using multiple browsers at the same time for the same connect instance is not supported, and causes issues with the rtc communication.
 
 
-### Downloading Streams with npm
+## Downloading Streams with npm
```

Comment:
The resultant outline is:

- Amazon Connect Streams Documentation
  - Overview
  - Architecture
  - Getting Started
    - Whitelisting
      - A few things to note:
  - Downloading Streams with npm
  - Importing Streams with npm and ES6
  - Usage with TypeScript
  - Downloading Streams from Github
  - Build your own with NPM
  - Using the AWS SDK and Streams
  - Initialization
    - `connect.core.initCCP()`
      - A few things to note:
    - `connect.core.terminate()`
  - Event Subscription
    - `connect.agent()`
    - `connect.contact()`
  - Agent API
  - Contact API
  - Connection API
  - VoiceConnection API
  - ChatConnection API
  - Utility Functions
  - Enumerations
  - Streams Logger
  - CCP Error Logging
  - Agent States Logging